src/common/webview.cpp

```diff
@@ -86,6 +87,24 @@ wxString wxWebViewConfiguration::GetDataPath() const
     return m_impl->GetDataPath();
 }
 
+// wxWebViewWindowFeatures
+wxWebViewWindowFeatures::wxWebViewWindowFeatures(wxWebView * childWebView):
+    m_childWebViewWasUsed(false),
+    m_childWebView(childWebView)
+{ }
+
+wxWebViewWindowFeatures::~wxWebViewWindowFeatures()
+{
+    if (m_childWebView && !m_childWebViewWasUsed)
+        delete m_childWebView;
```

Comment:
I wonder if we could avoid this by making `GetChildWebView()` reset the pointer to `nullptr` after returning it? In fact, what about using `unique_ptr<>` here too and returning it by value from `GetChildWebView()` to make it absolutely clear that the caller takes its ownership?

Reply:
Using a `std::unique_ptr` would probably make sense. But I intentionally wanted `GetChildWebView()` to be callable multiple times by application code. E.g. it could make sense to do something with the webview before processing the rest of `wxWebViewWindowFeatures` in another method.

As this is a control the application code shouldn't have to worry to much about ownership as that is more or less automatic once it's added to a window.